types/ember__test-helpers/index.d.ts

```diff
@@ -189,8 +189,21 @@ declare module '@ember/test-helpers/settled' {
 declare module '@ember/test-helpers/setup-context' {
     import Resolver from '@ember/application/resolver';
 
+    interface Owner {
+      lookup: (name: string) => unknown;
+      register: (name: string, mockService: unknown) => void;
+    }
+    export interface AppContext {
```

Comment:
There are other APIs that are public and added to the context. From `@ember/test-helpers` perspective, these types refer to public APIs:

https://github.com/emberjs/ember-test-helpers/blob/dd5fbb1280274068c046a8ff0a86177e9507e5a1/addon-test-support/%40ember/test-helpers/setup-context.ts#L21-L31

As well as the `element` property (when running `setupRenderingTest`/`setupApplicationTest` variants, but not `setupTest`), that is defined in these locations:

* https://github.com/emberjs/ember-test-helpers/blob/dd5fbb1280274068c046a8ff0a86177e9507e5a1/addon-test-support/%40ember/test-helpers/setup-rendering-context.ts#L258

* https://github.com/emberjs/ember-test-helpers/blob/dd5fbb1280274068c046a8ff0a86177e9507e5a1/addon-test-support/%40ember/test-helpers/setup-application-context.ts#L134-L138